website/www/site/content/en/documentation/patterns/shared-class.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Cache data using a shared object"
+---
+
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Cache data using a shared object
+
+A cache is a software component that stores data so that future requests for that data can be served faster. Side inputs, stateful `DoFn` and calling an external service can be used for accessing a cache. The Python SDK provide another option in the shared module, which can be more efficient than side inputs in terms of the amount of memory required and simpler than stateful `DoFn`.
```

Comment:
```suggestion
A cache is a software component that stores data so that future requests for that data can be served faster. To access a cache, you can use side inputs, stateful `DoFn`, and calling an external service. The Python SDK provides another option in the shared module. This option can be more memory-efficient than side inputs and simpler than a stateful `DoFn`.
```

Reply:
Resolved by a separate commit.